tests/Opus/LanguageTest.php

```diff
@@ -37,23 +37,31 @@
 
 namespace OpusTest;
 
+use Opus\Db2\Database;
 use Opus\Document;
 use Opus\File;
-use Opus\Language;
 use Opus\Licence;
 use Opus\Model\DbException;
+use Opus\Model2\Language;
 use OpusTest\TestAsset\TestCase;
 
 use function count;
 
 class LanguageTest extends TestCase
 {
+    public function tearDown()
+    {
+       // Due to the lack of a refresh/reset method for the entity manager we need to clear the language table
+       // TODO: Maybe the Codeception testing framework has a solution for a better handling of database cleanup.
+        $q = Database::getEntityManager()->createQuery("delete from " . Language::class);
```

Comment:
Ist dieser Code wirklich notwendig? Die Tests laufen auch ohne durch und die alte Bereinigung der Tabellen sollte ausreichen, da wir ja jetzt für jeden Test einen neuen EntityManager verwenden. Wenn das als erste Schritt für den Wechsel vom Cleanup in setUp zum Cleanup im Teardown gedacht war, dann würde ich das als Issue nach hinten schieben, bis Bedarf besteht. Ich glaube im Augenblick laufen die Tests ausreichend gut.